templates/pkgstatus.html

```diff
@@ -90,5 +90,5 @@
 {%- endmacro %}
 
 {% macro linkorigin(origin) -%}
-<a target="_new" data-toggle="tooltip" title="Portsmon for {{ origin }}" href="http://portsmon.freebsd.org/portoverview.py?category={{ origin.split('/')[0] }}&amp;portname={{ origin.split('/')[1] }}"><span class="glyphicon glyphicon-tasks"></span>{{ origin }}</a>
+<a target="_new" data-toggle="tooltip" title="Port skeleton of {{ origin }}@{{ build.buildname }}" href="https://cgit.freebsd.org/ports/tree/{{ origin }}?id={{ build.buildname }}"><span class="glyphicon glyphicon-tasks"></span>{{ origin }}</a>
```

Comment:
Need to add `build` as a parameter in the macro prototype and then add it into the `pkgstatus.linkorigin` call in templates/build.html

Reply:
@bdrewery Like this?